Gruntfile.js

```diff
@@ -1,20 +1,60 @@
+/*
+Copyright (c) 2020 The Gamepad Navigator Authors
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-lab/gamepad-navigator/raw/master/AUTHORS.md.
+
+Licensed under the BSD 3-Clause License. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the BSD 3-Clause License at
+https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
+*/
+
 /* eslint-env node */
 "use strict";
 
 module.exports = function (grunt) {
     grunt.config.init({
+        licenseBanner: grunt.file.read("templates/LICENSE-banner.txt"),
+        currentMarkdownBanner: /^(<!--)\s[\w\W]+?(-->)\s\s/m,
+        currentJsAndJson5Banner: /^\/\*\sCopyright \(c\)[\w\W]+?\s\*\/\s\s/m,
         lintAll: {
             sources: {
                 js:    ["./src/js/**/*.js", "tests/js/**/*.js", "./*.js"],
-                md:    [ "./*.md", "tests/**/*.md"],
+                md:    ["./*.md", "tests/**/*.md", "docs/*.md"],
                 json:  ["./*.json", "./.*.json", "tests/**/*.json"],
                 json5: ["./*.json5", "tests/**/*.json5"],
                 other: ["./.*"]
             }
+        },
+        usebanner: {
+            jsAndJson5: {
+                options: {
+                    position: "top",
```

Comment:
This and the other position variable can either be omitted (as they are the default) or moved to `usebanner.options` rather than repeated under both tasks.

Reply:
Thanks for the suggestion @the-t-in-rtf. I forgot that `top` is the default value for the banner, so I'll remove this.